lib/cli/index.js

```diff
@@ -6507,6 +6507,28 @@ export async function createMultiXBom(pathList, options) {
       parentComponent = parentComponent.components[0];
       delete parentComponent.components;
     }
+    // Add references between the multiple sub-boms
+    // Default is 'provides', but since some tools only generate a tree for 'dependsOn',
+    // this can be configured with an EnvVar
+    const multiBomComponentRef =
+      "dependsOn" === process.env.MULTI_BOM_COMPONENT_REF
+        ? "dependsOn"
+        : "provides";
```

Comment:
`provides` is only useful for CBOMs at this point? We can use default of `dependsOn`?

Reply:
I didn't realize that. I figured that logically it should not be a 'dependsOn' relation, my personal use however will be that every time because of the dependency-tree in Dependecz-Track. So yeah, I don't mind turning the default around!